### PR TITLE
[quant][graphmode] Enable tests

### PR DESCRIPTION
**Pull request by jerryzh168**
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #27119 [WIP][quant][graphmode] Prepack folding for conv2d
* #27118 [quant][graphmode] Add insert_prepack_unpack for conv2d
* **#27103 [quant][graphmode] Enable tests**
* #27102 [quant][graphmode][fix] insert_prepack_unpack pattern
* #27093 [quant][graphmode] API - add more passes to graph mode

Summary:
att

Test Plan:
python test/test_quantization.py 'GraphModePostTrainingQuantTest'

Reviewers:
pt1quant

Subscribers:

Tasks:

Tags:

Differential Revision: [D17678261](https://our.internmc.facebook.com/intern/diff/D17678261)

### CODE REVIEW

File: torch/quantization/_quantize_script.py [hunk 1]
```diff
@@ -66,11 +66,12 @@ def quantize_script(model, qconfig_dict, run_fn, run_args, inplace=False):
...
         raise ValueError('input script module does not have forward method')
+    assert not inplace, "We don't support inplace right now"
     if not inplace:
         model = model.copy()
     scripted_qconfig_dict = {k: script_qconfig(v) for k, v in qconfig_dict.items()}
     torch._C._jit_pass_fold_convbn(model._c)
     prepare_script(model, scripted_qconfig_dict, True)
     run_fn(model._c._get_method('forward'), *run_args)
-    convert_script(model, True)
+    model = convert_script(model, False)
```
Comment (ZolotukhinM) [inline torch/quantization/_quantize_script.py]: Please add a comment that this is temporary (and why).

File: test/test_quantization.py [hunk 2]
```diff
@@ -644,15 +644,18 @@ def checkQuantized(model):
...
-        r"""Quantize SingleLayerLinearModel which has one Linear module, make sure it is swapped
-        to nnq.Linear which is the quantized version of the module
+        r"""Compare the result of quantizing single linear layer in
+        eager mode and graph mode
         """
         # eager mode
-        model_eager = quantize(AnnotatedSingleLayerLinearModel(), test_only_eval_fn,
+        annotated_linear_model = AnnotatedSingleLayerLinearModel()
+        linear_model = SingleLayerLinearModel()
+        linear_model.fc1.weight = torch.nn.Parameter(annotated_linear_model.fc1.module.weight.detach())
```
Comment (ZolotukhinM) [inline test/test_quantization.py]: This is not a part of eager mode API, right? We should separate graph mode from eager mode and also comment each extra necessary step we need to make (i.e. it seems that we only need these lines in graph mode - we need to explain why).

**Review by ZolotukhinM — APPROVED** [review]
Looks good!

File: test/test_quantization.py (hunk 2 above)
Comment (jerryzh168) [reply test/test_quantization.py]: this is just making sure eager mode and graph mode models share the same parameters

**Pull request closed without merging**